packages/design-system/src/components/Table/Table.test.jsx

```diff
@@ -127,8 +122,6 @@ describe('Table', function () {
       const tableCaption = wrapper.find('TableCaption');
 
       expect(tableCaption.prop('_scrollableNotice')).toBe('foo scrollable notice');
-
```

Comment:
I think we should keep this

Reply:
Was there a particular reason why you deleted all these snapshots? I think some are probably not necessary but it doesnt hurt to keep them. Other tests definitely should have them, because they test for markup changes.

Reply:
Ah, I was thinking checking for the props/classes would be sufficient to cover the test, but yes, the snapshot can additionally catch the markup changes. Will add back.

Reply:
I think you can remove the snapshots for simple class changes, but for any tests that involve more markup changes or multiple attribute changes, we should add the snaps